### DISCUSSION TITLE
Home Assistant unable to play clips

### DISCUSSION THREAD

**Discussion by gradation0746** [blakeblackshear/frigate#6406] [Ask A Question]
I have installed Frigate via Docker (v. 0.12.0-DA3E197)

I have Home Assistant up and running, also via Docker (v. 2023.4.6)

I have a mqtt broker running, docker. Running correctly.

I have HACS/Frigate Integration/Lovelace front end all working.

Home Assistant is behind Nginx Proxy Manager. Frigate is accessible locally only.

I can view the live stream just fine. I navigate to Media -> Frigate -> Clips and click on a clip to open it and the player window stutters. If I inspect the player, the URL for the video is cycling through many different addresses. They all begin with the address for HA followed by a string of random letters and numbers. 

This happens in different browsers and I also cannot view the clip on my telephone (android).

Unsure if it is related but the error in the log from Home Assistant reads: Detected integration that calls media_source.async_resolve_media without passing an entity_id. Please report issue for media_source using this method at homeassistant/components/media_source/__init__.py, line 147: report("calls media_source.async_resolve_media without passing an entity_id")

I promise I've searched extensively for similar problems but cannot find related posts. Any and all help is thoroughly appreciated.

Edit - I can view the clip on my phone by tapping on the notification. The clip will not play if I navigate to it.

**Comment by blakeblackshear** [comment]
Are you certain you have Web Browser selected as the playback device when you are in the media browser?

**Reply by gradation0746** [reply]
Yes. In the lower right corner, "browser" is selected.

I actually can view the clips in the HA app on my phone (Android).

Obviously, this must be a browser issue. I am on Brave, v. 1.51.110
